app/forms/course_details_form.rb

```diff
@@ -97,6 +97,12 @@ def require_age_range?
     !trainee.early_years_route?
   end
 
+  def missing_fields
```

Comment:
This has been moved to the base form `TraineeForm` so i think we can get rid of this